apps/ledger-live-mobile/src/components/RootNavigator/BaseNavigator.tsx

```diff
@@ -351,8 +354,14 @@ export default function BaseNavigator() {
       />
       <Stack.Screen
         name={NavigatorName.Exchange}
-        component={ExchangeNavigator}
-        options={{ headerStyle: styles.headerNoShadow, headerLeft: null }}
+        component={
+          ptxSmartRouting.enabled ? ExchangeLiveAppNavigator : ExchangeNavigator
+        }
+        options={
+          ptxSmartRouting.enabled
+            ? { headerShown: false }
+            : { headerStyle: styles.headerNoShadow, headerLeft: null }
+        }
```

Comment:
Why do we create a new navigator instead of updating the `navigationParams` in the `FabActions` ?